test.py

```diff
@@ -1149,6 +1181,17 @@ async def shutdown(loop, signo, signaled):
         loop.add_signal_handler(signo, lambda: asyncio.create_task(shutdown(loop, signo, signaled)))
 
 
+def pattern_to_case(value: str) -> TestCase:
+    """From a command line test pattern create a TestCase."""
+    pattern = re.compile(r'^(?P<suite>[a-zA-Z0-9_-]*)(::(?P<test>[a-zA-Z0-9_-]+))?(::(?P<case>[a-zA-Z0-9_-]+))?$')
+    match = pattern.match(value)
+    if not match:
+        raise ValueError("Invalid pattern. Must be 'suite', 'suite::test', or 'suite::test::case'.")
```

Comment:
I was hoping wildcards would continue being supported. ./test.py raft is a convenient handle for running everything related to raft.

Reply:
Can you give an example of when such wildcards are useful to you? In my experience I always noticed the exact opposite - you type some word "xyz" and you get a few unrelated tests that happen to contain this word in the middle of the name.
That being said, we could always fall back to allowing a wildcard as well - if the test path doesn't have a "::" and isn't one of the small number of suite names - it is looked up as a substring search.